### PR TITLE
All all Monster Cards. Removed comments.

### PR DESCRIPTION
**Pull request by shweta3783**
Ignored these files in Sonar,Codecov and Travis config

### CODE REVIEW

**Comment by Rockie156** [comment]
Please refactor into a single folder (see issue on board)

Also do not update Binary files. I don't think it should be tracked (.coverage)

File: .travis.yml [hunk 1]
```diff
@@ -23,7 +23,6 @@ before_script:
 script:
   - pytest --cov-report term --cov=.
   - coverage xml
-  - npm --prefix $FRONTEND_DIR test
```
Comment (Rockie156) [inline .travis.yml]: why was this removed?

File: backend/src/main/game/monster/concrete_monster_cards/wild.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+from backend.src.main.game.monster.random_monster_card import RandomMonsterCard
+from backend.src.main.game.monster.values import DungeonCardValues, NumberedRoomTileValues
+
+
+class Wild(RandomMonsterCard):  # pylint: disable=too-few-public-methods
+
+    def __init__(self):
+        map_values = {NumberedRoomTileValues.ONE: DungeonCardValues.MONSTER,
```
Comment (Rockie156) [inline backend/src/main/game/monster/concrete_monster_cards/wild.py]: curly brace usage should be consistent, either same line or next line

Comment (shweta3783) [reply backend/src/main/game/monster/concrete_monster_cards/wild.py]: This has been resolved
